src/components/StudyPane/Header/Title.tsx

```diff
@@ -1,16 +1,19 @@
 'use client';
 
-import { ChangeEvent, useEffect, useRef, useState, useCallback } from "react";
+import { ChangeEvent, useContext, useEffect, useRef, useState, useCallback } from "react";
 import { updateStudyName } from '@/lib/actions';
 import { StudyData } from '@/lib/data';
+import { FormatContext } from '../index';
 
 
 const Title = ({ study }:{
-    study: StudyData,
+    study: StudyData;
```

Comment:
Minor: comma or semicolon is not needed there. You can either remove it or leave it as is.